### PR TITLE
[PAY-540] Add analytics to buy audio flow

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

Adds analytics in amplitude to the Buy Audio flow.

- Tracks the opening of the on ramp flow so we can see how many users make it through the 3rd-party onramp funnel, and how long it takes them
- Tracks the success/failure, with the surplus (or deficit) $AUDIO amount in the success case to measure how good our exchange accuracy is.

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Tested on local against prod, verified in amplitude

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

This is monitoring!

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-amp-onramp-sagas

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-amp-onramp-sagas

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-amp-onramp-sagas

File: packages/web/src/store/application/ui/buy-audio/sagas.ts [hunk 1]
```diff
@@ -627,15 +641,40 @@ function* startBuyAudioFlow({
...
+    const outputAmount = convertWAudioToWei(transferAmount)
     yield* put(
       increaseBalance({
-        amount: outputAmount
+        amount: weiToString(outputAmount)
+      })
+    )
+
+    // Record success
+    yield* put(
```
Comment (piazzatron) [inline packages/web/src/store/application/ui/buy-audio/sagas.ts:652]: Brutal conversions 😆 Maybe we should log these out also?

**Review by piazzatron — APPROVED** [review]
Looks great, apologies for delay!

File: packages/web/src/store/application/ui/buy-audio/sagas.ts (hunk 1 above)
Comment (rickyrombo) [reply packages/web/src/store/application/ui/buy-audio/sagas.ts:652]: I think we'll get the result from what's in Amplitude :)

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-amp-onramp-sagas

**Pull request merged**